src/taskgraph/util/vcs.py

```diff
@@ -157,6 +157,16 @@ def find_latest_common_revision(self, base_ref):
             "{node}",
         ).strip()
 
+    def does_revision_exist_locally(self, revision):
```

Comment:
Please add this to the abstract base class with docstring

Reply:
Nice catch! Done.